ModPatches/Medieval Overhaul/Patches/Medieval Overhaul/ThingDefs_Races/MO_CE_Patch_Daer.xml

```diff
@@ -104,4 +118,27 @@
 		</value>
 	</Operation>
 
+	<Operation Class="PatchOperationConditional">
+		<xpath>Defs/ThingDef[defName="DankPyon_Daer"]/comps</xpath>
+		<nomatch Class="PatchOperationAdd">
+			<xpath>Defs/ThingDef[defName="DankPyon_Daer"]</xpath>
+			<value>
+				<comps />
+			</value>
+		</nomatch>
+	</Operation>
+
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="DankPyon_Daer"]/comps</xpath>
+		<value>
+			<li Class="CombatExtended.CompProperties_ArmorDurability">
+				<Durability>3000</Durability>
```

Comment:
Please confirm the armor durability values are correct; the standard formula is 250 * (`baseBodySize` + `baseHealthScale`). I end up with 250 * (4.2 + 9.0) = 3300.